### ISSUE TITLE
debugging sort()

### ISSUE THREAD

**Issue by gavinking** [ceylon/ceylon-ide-eclipse#1261]
Try to debug this function:

```ceylon
shared void test() {
    print("starting");
    print({1,-1,3,-2,-5,2,0}.sort(increasing));
    print({1,-1,3,-2,-5,2,0}.sort(decreasing));
}
```

Crazily, it's possible to step into `test_$2.<init>` but it's not possible to step into `sort()`!

That's backwards, I think.

**Comment by davidfestal** [comment]
I'm looking into it.

**Comment by gavinking** [comment]
Thanks

**Comment by davidfestal** [comment]
Fixed by ac139c170b6e88573a4a09cbf74c7b1ae25f9d31

**Comment by gavinking** [comment]
Thanks!
